components/nav/NamespaceFilter.vue

```diff
@@ -180,7 +180,10 @@ export default {
 
     value: {
       get() {
-        const values = this.$store.getters['prefs/get'](NAMESPACE_FILTERS);
+        const prefs = this.$store.getters['prefs/get'](NAMESPACE_FILTERS);
+        const clusterId = this.$store.getters['clusterId'];
+        const values = prefs[clusterId];
```

Comment:
Does this work if I have an old pref which is just a string?

Reply:
Good catch, updated!